arbitrum-docs/how-arbitrum-works/02-transaction-lifecycle.mdx

```diff
@@ -1,4 +1,26 @@
-# Overview: The Lifecycle of an Arbitrum Transaction
+---
+title: Transaction Lifecycle
+---
```

Comment:
This should probably be a complete front matter.

Reply:
Adding front matter for all 👍